svgs/allWearables.ts

```diff
@@ -0,0 +1,161 @@
+import { SleeveObject, ItemTypeInputNew } from "../scripts/itemTypeHelpers";
+import {
+  bodyWearable,
+  BodyWearableOutput,
+  wearable,
+} from "../scripts/svgHelperFunctions";
+import {
+  wearablesLeftSvgs,
+  wearablesRightSvgs,
+  wearablesBackSvgs,
+} from "../svgs/wearables-sides";
+import { sleeveSvgs } from "./wearables";
+import { itemTypes } from "../data/itemTypes/itemTypes";
+import { allSleeves } from "./wearables";
+import { allBadges } from "./BadgeData";
+import { getBaadge } from "./allBadges";
+
+//for wearables that have irregular file names
+const svgExceptons = [
```

Comment:
could you combine this array with the object below in a mapping?

Reply:
done